explore/src/main/scala/explore/targeteditor/SiderealTargetEditor.scala

```diff
@@ -122,6 +138,30 @@ object SiderealTargetEditor:
     attemptCombine(ra, dec)
       .map(ProperMotion.apply.tupled)
 
+  // An UndoSetter that doesn't really update any undo stacks
+  private def noopUndoSetter[M](view: View[M]): UndoSetter[M] =
```

Comment:
This seems to work and be useful here. I don't know if it is generally useful enough to put somewhere like the UndoSetter companion object?